pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -154,9 +154,7 @@ func (e *kube) SetupWorkflow(ctx context.Context, conf *types.Config, taskUUID s
 					return err
 				}
 				log.Trace().Str("pod-name", stepName).Msgf("Creating service: %s", step.Name)
-				// TODO: support ports setting
-				// svc, err := Service(e.config.Namespace, step.Name, stepName, step.Ports)
-				svc, err := Service(e.config.Namespace, step.Name, stepName, []string{})
+				svc, err := Service(e.config.Namespace, step.Name, step.Alias, step.Ports)
```

Comment:
Was it intended to use `step.Alias` instead of `stepName` while `stepName` is now only used in the trace log?

Reply:
> stepName is now only used in the trace log

No, it becomes `metadata.name` field - `wp-01hdq6gbkw1mn6k1655fs3rntf-0-services-0` in the example.

> Was it intended to use step.Alias instead of stepName

Yes, it becomes `spec.selector.step` field - `database` in the example.